app/screens/transfer-amount-input-screen/transfer-amount-input-screen.tsx

```diff
@@ -98,6 +96,12 @@ export class TransferAmountInputScreen extends React.Component<TransferAmountInp
       errorMessage,
       isCreatingTx,
     } = this.props.txStore
+
+    const skipToConfirm = this.props.navigation.getParam("skipToConfirm")
+    if (skipToConfirm) {
+      return <LoadingScreen />
```

Comment:
on second thought, this if did not handle the case where amount is not set, which is check above with `amount && skipToConfirm`

Reply:
also if `createTransactionForSigning` throws error